packages/bsky/src/auth.ts

```diff
@@ -30,6 +30,45 @@ export const authOptionalVerifier =
     return authVerifier(idResolver, opts)(reqCtx)
   }
 
+export const authOptionalAccessOrRoleVerifier = (
```

Comment:
This is basically a ported over version of the existing role/access check from pds but it makes the check optional